app/src/main/java/org/mozilla/focus/search/SearchEngineParser.java

```diff
@@ -39,7 +38,7 @@ public static SearchEngine load(AssetManager assetManager, String identifier, St
         }
     }
 
-    @VisibleForTesting static SearchEngine load(String identifier, InputStream stream) throws IOException, XmlPullParserException {
+    public static SearchEngine load(String identifier, InputStream stream) throws IOException, XmlPullParserException {
```

Comment:
Does this need to be public? SearchEngineManager is in the same package and should have access already.

Reply:
Ah, I guess not! There was an AS error highlight, but I guess technically it's fine. Will change it back.